app/views/gang.html

```diff
@@ -26,6 +25,17 @@ <h2 class="heading-medium">Known nominals:</h2>
         <p>none</p>
       {% endif %}
 
+      <h2 class="heading-medium">Known tensions:</h2>
+      {% if tensions.length %}
+        <ul class="list list-bullet">
+          {% for tension in tensions %}
+          <li><a href="/gang/{{tension.gang.index}}">{{tension.gang.name}}</a> - Threat level: {{tension.threat}}</li>
```

Comment:
Agreed - threat level was placeholder. Will change to 'tension level'.

OK to stick with high, medium, low at the moment?